### PR TITLE
Added support for host_name

### PR DESCRIPTION
**Pull request by prajwalpatil25**
# Description
Added host_name support under host group resource.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [X] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [X] I have verified that new and existing unit tests pass locally with my changes
- [X] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [X] Acceptance tests
image

### CODE REVIEW

File: powerstore/resource_host_group.go [hunk 1]
```diff
@@ -68,7 +69,21 @@ func (r *resourceHostGroup) Schema(ctx context.Context, req resource.SchemaReque
 
 			"host_ids": schema.SetAttribute{
 				ElementType:         types.StringType,
-				Required:            true,
+				Optional:            true,
+				Computed:            true,
+				Description:         "The list of hosts to include in the host group.",
```
Comment (shenda1) [inline powerstore/resource_host_group.go]: The list of host IDs to include in the host group

File: powerstore/resource_host_group.go [hunk 2]
```diff
@@ -68,7 +69,21 @@ func (r *resourceHostGroup) Schema(ctx context.Context, req resource.SchemaReque
...
+				Validators: []validator.Set{
+					setvalidator.ConflictsWith(path.Expressions{
+						path.MatchRoot("host_names"),
+					}...),
+				},
+			},
+
+			"host_names": schema.SetAttribute{
+				ElementType:         types.StringType,
+				Optional:            true,
```
Comment (shenda1) [inline powerstore/resource_host_group.go:85]: The list of host names to include in the host group

File: powerstore/resource_host_group.go (hunk 1 above)
Comment (prajwalpatil25) [reply powerstore/resource_host_group.go]: done

File: powerstore/resource_host_group.go (hunk 2 above)
Comment (prajwalpatil25) [reply powerstore/resource_host_group.go:85]: done

**Review by shenda1 — DISMISSED** [review]

**Review by shenda1 — DISMISSED** [review]

**Comment by Sakshi-dell** [comment]
Please add the current coverage of the file.
changes LGTM

**Review by Sakshi-dell — DISMISSED** [review]

**Review by Sakshi-dell — APPROVED** [review]

**Review by shenda1 — APPROVED** [review]

**Pull request merged**